### DISCUSSION TITLE
Is there a way to make the bsp layout stack on the right?

### DISCUSSION THREAD

**Discussion by bustinbung** [asmvik/yabai#1914] [Q&A]
I know there are commands for mirroring the window tree `yabai -m space --mirror y-axis`, but I want yabai to tile the windows to the right by default, instead of on the left.

Like this:
```
+------+------+
|      |      |
|      +------+
|      |      |
+------+------+
```

Not this:
```
+------+------+
|      |      |
+------+      |
|      |      |
+------+------+
```

I haven't been able to find a way to determine if the window tree is mirrored or not through `yabai -m query`, and I don't think that determining this based off of window sizes is elegant or handles edge cases well.

I've tried including the mirror command in my config, but the issue comes when I reload yabai: it mirrors again, flipping it again and reverting to left stack.

Any ideas?

**Comment by Bellavene** [comment]
Read the documentation.
`window_placement second_child`

**Reply by bustinbung** [reply]
I've already set that value in my config, I believe it's actually set that way in the default config. The issue is not that yabai doesn't end up tiling the windows that way, it's that it doesn't do it automatically when I switch between float and bsp layouts or restart yabai sometimes.

**Comment by Tenshikun** [comment]
Hey, I've found this (replace the key bindings with those you want, and the application obv) :
`cmd + shift - return : yabai -m window --focus east; yabai -m window --insert south && open -a Terminal ~; `

**Reply by Tenshikun** [reply]
EDIT : works only if there is at least one window so I have to improve it with a test condition with yabail -m query
